### PR TITLE
Akodali/fix artifact name

### PR DESCRIPTION
**Pull request by akodali18**
Just fixing the artifact name.
Some trailing spaces seem to be deleted by my editor , hope thats fine.

### CODE REVIEW

File: pom.xml
Comment (spramanik4667) [inline pom.xml]: Should it be reverted back to 0.9.0-SNAPSHOT.

Comment (haosong) [reply pom.xml]: I think the version here is for the next development version, so it should be `0.9.1-SNAPSHOT`

Comment (akodali18) [reply pom.xml]: I reverted this back to 0.9.0 given we are changing the artifact name

**Review by haosong — APPROVED** [review]
LGTM, if this is just for fixing the artifact name. 
* Or we could wait for the license file to merge together.

**Review by spramanik4667 — APPROVED** [review]

**Pull request merged**
